libmscore/input.h

```diff
@@ -79,8 +88,11 @@ class InputState {
       int string() const                  { return _string;             }
       void setString(int val)             { _string = val;              }
 
-      bool repitchMode() const            { return _repitchMode;    }
-      void setRepitchMode(bool val)       { _repitchMode = val;     }
+      bool repitchMode() const            { return _noteEntryMethod == NoteEntryMethod::REPITCH;    }
+      void setRepitchMode(bool val)       { if(val)
+                                                setNoteEntryMethod(NoteEntryMethod::REPITCH);
+                                            else
+                                                setNoteEntryMethod(NoteEntryMethod::STEPTIME);}
```

Comment:
`repitchMode()` --> `noteEntryMethod() == NoteEntryMethods::REPITCH`would be a regression according to me. A bit less for the other, but really, if you can encapsulate the code, I would keep repitchMode and setRepitchMode.